sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -26,13 +26,13 @@ To use this starter in an web application, please add following packages:
     <version>3.0.0-beta.1</version>
 </dependency>
 <dependency>
-    <groupId>org.springframework.security</groupId>
-    <artifactId>spring-security-oauth2-client</artifactId>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-oauth2-client</artifactId>
```

Comment:
@yiliuTo , 
(cc: @saragluna , @backwind1233 , FYI)

we should use `xxx-starter` instead of `xxx`, because `xxx-starter` can include other necessary dependencies. Like `spring-security-oauth2-jose`.

Reply:
FYI @wujack778.